### PR TITLE
make --strict also a build option

### PR DESCRIPTION
**Pull request by boegel**
@wpoely86: please review

Useful for https://github.com/hpcugent/easybuild-easyblocks/pull/622

### CODE REVIEW

**Comment by wpoely86** [comment]
I'm probably missing something here but what exactly does this change?

**Comment by boegel** [comment]
@wpoely86: it enables using `build_option('strict')` without running into a `KeyError`

File: easybuild/tools/config.py [hunk 1]
```diff
@@ -119,6 +121,9 @@ def mk_full_default_path(name, prefix=DEFAULT_PREFIX):
     True: [
         'cleanup_builddir',
     ],
+    DEFAULT_STRICT: [
+        'strict',
+    ],
```
Comment (boegel) [inline easybuild/tools/config.py]: @wpoely86: key change is here ^

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**